src/arcade/Documentation/UnifiedBuild/Unified-Build-Controls.md

```diff
@@ -142,8 +142,9 @@ In addition to these default high level controls, there may be additional compon
 | TargetOS | Same as `BuildOS` | `BuildOS` | The operating system of the machine that will run the binary -> the end user’s machine. |
 | HostOS | Same as `BuildOS` | `TargetOS` | The operating system of the machine that will run the produced tool (i.e. compiler) to generate the binary for the target operating system. |
 | BuildRid | Valid RIDs | RID of the the currently executing runtime | The RID of the runtime that is running the build |
-| TargetRid | Valid RIDs | When building non-portable, the OS of build Rid + TargetArchitecture. When building portable, `TargetOS-TargetArchitecture`. | The RID of the runtime that will run the binary -> the end user’s machine. |
-| HostRid | Valid RIDs | `TargetRid` | The RID of the runtime that will run the produced tool (i.e. compiler) to generate the binary for the target operating system. |
+| TargetRid | Valid RIDs or custom RID | When building non-portable, the OS of build Rid + TargetArchitecture. When building portable, `TargetOS-TargetArchitecture`. | The RID of the runtime that will run the binary -> the end user’s machine. |
+| HostRid | Valid RIDs or `TargetRid` | `TargetRid` | The RID of the runtime that will run the produced tool (i.e. compiler) to generate the binary for the target operating system. |
+| BaseRid | Valid RIDs | OS portion of `NETCoreSdkPortableRuntimeIdentifier` appended with `-TargetArchitecture` | A known RID to use as a parent of a custom RID specified in `TargetRid` if `TargetRid` is unknown. |
```

Comment:
@jkoritzinsky @ViktorHofer @akoeplinger what are your thoughts on renaming `BaseRid` to `PortableTargetRid`.

Before .NET 10, `BaseOS` was a property that only existed to pass info from the vmr's `runtime.proj` to runtime's `SourceBuild.props`. Now that it gets used more widely, perhaps we should give it a more descriptive name?

Reply:
Sounds like a good idea to me. I prefer the gnu configure terms.

Reply:
I'll look into renaming this next week.

Reply:
I agree that changing the name would make it more clear.

Reply:
@ViktorHofer I need changes in several repos for this. Can I make them in `dotnet/dotnet` and will those flow back to the upstream repos?

I didn't know there was syncing in that direction until I looked closer at this PR.